switch_model/wecc/get_inputs/cli.py

```diff
@@ -21,23 +23,64 @@ def main():
         config.yaml specifies the scenario parameters.
         The environment variable DB_URL specifies the url to connect to the database. """,
     )
-    parser.add_argument("--skip-cf", default=False, action='store_true',
-                        help="Skip creation variable_capacity_factors.csv. Useful when debugging and one doesn't"
-                             "want to wait for the command.")
-    parser.add_argument("--post-process", default=None, help="Run only this post process step.")
-    parser.add_argument("--overwrite", default=False, action='store_true',
-                        help="Overwrite previous input files without prompting to confirm.")
+    parser.add_argument(
+        "--skip-cf",
+        default=False,
+        action="store_true",
+        help="Skip creation variable_capacity_factors.csv. Useful when debugging and one doesn't"
+        "want to wait for the command.",
+    )
+    parser.add_argument(
+        "--post-process-only",
+        default=False,
+        action="store_true",
+        help="Run only post process steps.",
+    )
+    parser.add_argument(
+        "--post-process-step", default=None, help="Run only this post process step."
+    )
+    parser.add_argument(
+        "--overwrite",
+        default=False,
+        action="store_true",
+        help="Overwrite previous input files without prompting to confirm.",
+    )
     args = parser.parse_args()  # Makes switch get_inputs --help works
 
     # Load values from config.yaml
     full_config = load_config()
     switch_to_input_dir(full_config, overwrite=args.overwrite)
 
-    if args.post_process is None:
+    if not args.post_process_only and args.post_process_step is None:
         query_db(full_config, skip_cf=args.skip_cf)
-    print("Post-processing...")
-    run_post_process(full_config, step_name=args.post_process)
-    print(f"\nScript took {timer.step_time_as_str()} seconds to build input tables.")
+
+    print("\nRunning post processing...")
+
+    # Get location of post process scripts
+    post_process_path = ".".join(__name__.split(".")[:-1]) + ".post_process_steps"
+
+    def run_post_process(module):
+        """ Run a function from a given module """
+
+        # This uses python module syntax with a dot. Example: import foo.bar.test
+        mod = importlib.import_module(f".{module}", post_process_path)
+
+        post_process = getattr(mod, "post_process")
+
+        # Get specific configuration for the post process if specified
+        post_config = full_config.get(module, None)
+
+        # Run post process
+        post_process(full_config, post_config)
```

Comment:
it would make sense to me to only pass the config for that step to the process simplyfing this to one argument only